wrappers/python/setup.py

```diff
@@ -192,13 +192,17 @@ def buildKeywordDictionary(major_version_num=MAJOR_VERSION_NUM,
         if platform.system() == 'Darwin':
             extra_compile_args += ['-stdlib=libc++', '-mmacosx-version-min=10.7']
             extra_link_args += ['-stdlib=libc++', '-mmacosx-version-min=10.7', '-Wl', '-rpath', openmm_lib_path]
+            if 'MACOSX_DEPLOYMENT_TARGET' not in os.environ and platform.processor() != 'arm':
+                extra_compile_args += ['-mmacosx-version-min=10.7']
+                extra_link_args += ['-mmacosx-version-min=10.7']
```

Comment:
`-mmacosx-version-min=10.7` is already present in both lists, from the lines immediately above.  Did you mean to remove them from those lines?  Otherwise, I don't think the added lines are doing anything.